db/migrate/20230125194631_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :Title
+      t.text :Text
+      t.string :comments_counter
+      t.string :likes_counter
```

Comment:
- [ ] The type for comments_counter and likes_counter is expected to be an integer as there will be an increment operation that will happen as a user comments on a post or likes a post. Kindly work on these model types 👍🏽

Reply:
changed data type of comments_counter andlikes_counter to integer